### PR TITLE
Add boxel-component blueprint

### PR DESCRIPTION
**Pull request by lukemelia**
This blueprint allows a developer to generate a new boxel component (index.gts), a
CSS file, a usage.gts file, and an integration test. Run it using
`ember generate boxel-component your-component-name`.

### CODE REVIEW

File: packages/boxel/.eslintrc.js [hunk 1]
```diff
@@ -56,7 +57,7 @@ module.exports = {
         {},
         require('eslint-plugin-node').configs.recommended.rules,
         {
-          // add your custom rules and overrides for node files here
+          'node/no-unpublished-require': 'off',
```
Comment (backspace) [inline packages/boxel/.eslintrc.js:60]: Could this be applied more narrowly?

**Review by backspace — APPROVED** [review]
🤩

File: packages/boxel/.eslintrc.js (hunk 1 above)
Comment (lukemelia) [reply packages/boxel/.eslintrc.js:60]: Probably, but we disable this in our other packages, so the consistency seems like a good move.

**Review by danibonilha — APPROVED** [review]
This is awesome 🤩

**Pull request merged**